text/0811-element-modifiers.md

```diff
@@ -0,0 +1,232 @@
+---
+Stage: Accepted
+Start Date: 2022-03-29
+Release Date: Unreleased
+Release Versions:
+  ember-source: vX.Y.Z
+  ember-data: vX.Y.Z
+Relevant Team(s): Ember CLI, Learning
+RFC PR: https://github.com/emberjs/rfcs/pull/811
+---
+
+<!--- 
+Directions for above: 
+
+Stage: Leave as is
+Start Date: Fill in with today's date, YYYY-MM-DD
+Release Date: Leave as is
+Release Versions: Leave as is
+Relevant Team(s): Fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies
+RFC PR: Fill this in with the URL for the Proposal RFC PR
+-->
+
+# Element Modifiers
+
+## Summary
+
+This RFC introduces the concept of user defined element modifiers and proposes
+adding [ember-modifier](https://github.com/ember-modifier/ember-modifier)
+to the blueprints that back `ember new` and `ember addon`, providing an
```

Comment:
> and `ember addon`

It's not super clear if this means that `ember-modifier` will become a direct dependency of a new addon or if `ember-modifier` will become a dev dependency of a new addon's dummy app. I'm assuming it's the latter?

Reply:
@sergeastapov as @bertdeblock notes, we should clarify this. I believe it will be added to `devDependencies` of apps and `dependencies` of addons, per the usual (same as e.g. `@glimmer/component`.

Reply:
Are you sure about adding it to `dependencies` for addons? Most addons don't ship modifiers, so I would prefer to keep it under `devDependencies`. Both `@glimmer/*` packages are also included under `devDependencies`.

Reply:
Ah, that's a good point. Should be in `devDependencies` of both.

Reply:
thanks @bertdeblock and @chriskrycho! This was clarified in the "Motivation" section:

> This RFC seeks to fill this gap in Ember.js' development mental model by providing `ember-modifier` in the blueprint. `ember-modifier` will be added to `devDependencies`, same as e.g. @glimmer/component.